dll/win32/imm32/imm.c

```diff
@@ -5066,6 +5066,15 @@ ImmGetImeInfoEx(PIMEINFOEX pImeInfoEx,
     return NtUserGetImeInfoEx(pImeInfoEx, SearchType);
 }
 
+/***********************************************************************
+ *              ImmWINNLSGetIMEHotkey (IMM32.@)
+ */
+UINT WINAPI ImmWINNLSGetIMEHotkey(HWND hwndIme)
+{
+    TRACE("(%p)\n", hwndIme);
+    return 0;
```

Comment:
OK actually if you know since which Windows version this function just returns 0, you should mention this there, so that we know it's intentional, and is not a "stub to be implemented later in ReactOS" 😄

Reply:
Comment added.

Reply:
Comment added.